README.md

```diff
@@ -0,0 +1,84 @@
+![logo](http://springroll.io/assets/images/logo.png)
+
+# SpringRoll
+
+
+SpringRoll is a light-weight toolset for building accessible HTML5 games. The latest version, SpringRoll 2 is a large departure from its predecessor in that it provides less structure for game development while instead focusing on utilities to help make games more accessible and deployable at scale.
```

Comment:
Suggested edit: 

The latest version, SpringRoll 2, is a large departure from its predecessor. SpringRoll 2 is less prescriptive and contains only a few guidelines and restrictions. Instead, the framework focuses on utilities to help developers make games more accessible and deployable at scale.